RealEstate/apps/core/templates/core/categories.html

```diff
@@ -141,13 +103,13 @@ <h2 class="modal-title">Edit your category!</h2>
     <div class="modal-dialog">
     <div class="modal-content">
         <div class="modal-header">
-            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <button type="button" class=xr"close" data-dismiss="modal">&times;</button>
```

Comment:
Not sure what the "xr" is for. But this does remind me that this is another button type that shouldn't have padding. So in the style, we'll also need 
```button.close{margin:auto;}```